src/tests/help-flag/expected-stdout

```diff
@@ -12,10 +13,12 @@ Available Commands:
   init        Init directory and perform the first pull
 
 Flags:
-  -d, --dir string        use other working directory
-  -h, --help              print help for command
-  -l, --log-file string   path to a log file for details
-  -v, --verbose           print details
-      --version           version for bin_func_tests
+  -h, --help                       print help for command
+  -l, --log-file string            path to a log file for details
+  -H, --storage-api-host string    storage API host, eg. "connection.keboola.com"
```

Comment:
Zjednotil som nazvy `env` premennych podla design docs, a nazvy flags, ... aby sa to nemuselo rucne definovat.